resources/lang/en/admin.php

```diff
@@ -43,6 +43,8 @@
     'submission_extra_info' => 'Anything else we should know about this business?',
     'submission_status' => 'Status',
 
+    'website_invalid' => 'This website is invalid.  It must be between 5 and 100 characters, contain a domain, and must be accessible without redirection.  Watch out for using \'http\' when the website redirects to \'https\'.',
```

Comment:
This doesn't seem to be used?

Reply:
Yes - I intended to use it and then found there was a useful error from the server.  I've repurposed it as the intro text you suggested.